carmen-salas/exercises/answers.js

```diff
@@ -0,0 +1,72 @@
+// Question 1: Looping a Triange
+// 1.  **Looping A Triangle** 
+  // Write a loop that makes seven calls to console.log to output the following triangle:
+
+function triangle(){
+  let hash = '';
+
+  for (let i = 0; i < 7; i++) {
+      hash = hash + '#';
+      console.log(hash);
+  }
+}
+
+
+// Question 2: FizzBuzz
+function fizzBuzz() {
```

Comment:
Nice use of the `do... while` loop here. I love the use of the nested `if` statement. It's intuitive.

Feedback:
1. add a space between your operators!
i.e. Change `if((num%3)==0)` to `if ((num % 3) === 0)`. It's more readable that way.
2. Use the _strict equality operator_ for comparison (`===`). Unless we have a good reason, we don't want to expose our code to the type coercion that comes along with the abstract equality operator (`==`).